src/fmdb/FMDatabaseAdditions.h

```diff
@@ -166,7 +166,7 @@ NS_ASSUME_NONNULL_BEGIN
  @see [table_info](http://www.sqlite.org/pragma.html#pragma_table_info)
  */
 
-- (FMResultSet*)getTableSchema:(NSString*)tableName;
+- (FMResultSet * _Nullable)getTableSchema:(NSString*)tableName;
```

Comment:
For the spacing / formatting of the return type, I referenced other functions that have nullability annotations such as this one:

https://github.com/ccgus/fmdb/blob/240667c4d47b438458b917424150bc07e8c644b1/src/fmdb/FMDatabase.h#L608